kernel/src/process/process/mod.rs

```diff
@@ -92,6 +92,9 @@ pub struct Process {
     /// According to POSIX.1, the nice value is a per-process attribute,
     /// the threads in a process should share a nice value.
     nice: AtomicNice,
+    /// The adjustment value of the out-of-memory (OOM) killer score.
+    // FIXME: Support OOM killer.
+    oom_score_adj: AtomicI16,
```

Comment:
I think this file should be named as `oom_score` because it means the OOM score for the process.

On the other hand, `oom_score_adj` means a file that can be used to adjust the OOM score, right?